operator/midstream/iib.json

```diff
@@ -11,4 +11,4 @@
         "channel":"rhacs-1.0"
     },
     "support_url":"https://install.stackrox.io/collector/support-packages/ppc64le/2.5.0/support-pkg-2.5.0-latest.zip"
```

Comment:
@jdao-rh  we would need P/Z specific handling here for the support package URL cc @mdafsanhossain 
may be use `REMOTE_CLUSTER_ARCH` to construct the URL

Reply:
Yes, I will add that.